src/lib/viewers/image/ImageViewer.js

```diff
@@ -389,6 +411,15 @@ class ImageViewer extends ImageBaseViewer {
         if (this.isMobile) {
             this.imageEl.addEventListener('orientationchange', this.handleOrientationChange);
         }
+
+        if (this.options.enableAnnotationsImageDiscoverability) {
+            this.addListener('zoom', ({ newScale }) => {
```

Comment:
Instead of an anonymous function, should we just have `handleZoom` as a named function on the `ImageViewer` so that it can also be cleaned up in `destroy`?

Reply:
I can have it be a named function instead. I'm not seeing a `destroy` method within ImageViewer though.